docs/source/notebooks/regridding.ipynb

```diff
@@ -1114,17 +1156,15 @@
    "cell_type": "code",
    "execution_count": 7,
    "id": "21db46ab",
-   "metadata": {},
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-11-28T19:25:19.470218Z",
+     "iopub.status.busy": "2022-11-28T19:25:19.469646Z",
+     "iopub.status.idle": "2022-11-28T19:25:22.643114Z",
+     "shell.execute_reply": "2022-11-28T19:25:22.642032Z"
+    }
+   },
    "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "hwloc/linux: Ignoring PCI device with non-16bit domain.\n",
```

Comment:
@tlogan2000 this was the stderr that tripped Jenkins.  It's weird Jenkins do not have this but when we "restart kernel and run all cells" we have this.